### PR TITLE
Feature/issue1

### LINKED ISSUE #1: Change the value of N value to 1
add the value of n =1

### PR DESCRIPTION
**Pull request by Thna17**
add 1 #1

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 def main(n):
-    return n
+    return n
```
Comment (Chhunkea) [inline main.py:2]: noob add

File: main.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 def main(n):
-    return n
+    return n 
 
-n = 0
-print(main(n))
\ No newline at end of file
+n = 1
+
+print(main(n))
```
Comment (Chhunkea) [inline main.py:6]: print (*2)??????

**Review by Chhunkea — COMMENTED** [review]
Noob input

**Review by Chhunkea — APPROVED** [review]

**Comment by Chhunkea** [comment]
> add 1 #1

why so noob?

**Comment by Thna17** [comment]
NOOP TOO

**Review by Chhunkea — APPROVED** [review]

**Review by Chhunkea — APPROVED** [review]
noob

**Pull request closed without merging**